operate/webapp/src/main/java/io/camunda/operate/webapp/rest/DecisionInstanceRestService.java

```diff
@@ -58,10 +58,8 @@ public DecisionInstanceListResponseDto queryDecisionInstances(
   @GetMapping("/{decisionInstanceId}")
   public DecisionInstanceDto queryDecisionInstanceById(
       @PathVariable final String decisionInstanceId) {
-    final DecisionInstanceDto decisionInstanceDto =
-        decisionInstanceReader.getDecisionInstance(decisionInstanceId);
-    checkIdentityReadPermission(decisionInstanceDto);
-    return decisionInstanceDto;
+    checkIdentityReadPermission(decisionInstanceId);
+    return decisionInstanceReader.getDecisionInstance(decisionInstanceId);
```

Comment:
❓ With this change we are now retrieving the decision instance twice when authorizations are enabled, once in the `checkIdentityReadPermission` method and then again in the `getDecisionInstance` call, I'm not sure on the throughput/performance of this endpoint but it would be more efficient to use the older pattern (retrieve and then check)